client.go

```diff
@@ -113,7 +123,7 @@ func NewClient(nodes string) (*Client, error) {
 // the given server endpoint, using a specific remote API version.
 func NewVersionedClient(nodestring string, apiVersionString string) (*Client, error) {
 	nodes := strings.Split(nodestring, ",")
-	d, err := netutil.NewMultiDialer(nodes, nil)
+	addresses, err := netutil.AddressesFromNodes(nodes)
```

Comment:
We should probably have this fella (uniformly) randomise the order of the returned nodes - that way the first address in the list doesn't always get hammered by everyone.